### PR TITLE
Add docs: Why not PDB files?

### PR DESCRIPTION
**Pull request by uilianries**
I've added a better description for https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#kb-h017-pdb-files-not-allowed. Also, I included a small typo fix (thanks to the code editor).

### CODE REVIEW

**Comment by jgsogo** [comment]
@uilianries , please, fix the conflict. Thanks!

**Comment by uilianries** [comment]
@jgsogo Done!

File: docs/faqs.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Unless they are a general and extended utility in recipes (in which case, we sho
...
+The notation shown below is used for publishing packages where the original library does not make official releases. Thus, we use a datestamp corresponding to the date of a commit. In order to create reproducible builds, we also "commit-lock" to the latest commit on that day. Otherwise, users would get inconsistent results over time when rebuilding the package. An example of this is the [NanoRange](https://github.com/tcbrindle/NanoRange) library, where its package reference is `nanorange/20191001` and its sources are locked the latest commit on that date: https://github.com/conan-io/conan-center-index/blob/master/recipes/nanorange/all/conandata.yml#L2L3
```
Comment (jgsogo) [inline docs/faqs.md:66]: I think this PR shouldn't modify this FAQ.

Comment (uilianries) [reply docs/faqs.md:66]: What?! Probably I mixed the branches! I gonna fix it, thanks

**Comment by uilianries** [comment]
@jgsogo Done!

File: docs/faqs.md [hunk 2]
```diff
@@ -76,10 +76,20 @@ As described in the [Supported platforms and configurations](https://github.com/
...
+
+However, there are ways to get around this, one of them is through the [/Z7](https://docs.microsoft.com/en-us/cpp/build/reference/z7-zi-zi-debug-information-format) compilation flag, which can be passed through [environment variables](https://docs.microsoft.com/en-us/cpp/build/reference/cl-environment-variables). You can use your [profile](https://docs.conan.io/en/latest/reference/profiles.html#package-settings-and-env-vars) to customize your compiler command line.
+
+#### Why is there no option for PDB, as there is for fPIC?
+
+Adding one more common option, it seems the most simple and obvious solution, but it contains a side effect already seen with fPIC. It is necessary to manage the entire recipe, it has become a Boilerplate. So, adding PDB would be one more point to be reviewed for each recipe. In addition, in the future new options could arise, such as sanity or benchmark, further inflating the recipes. For this reason, a new option will not be added. However, the inclusion of the PDB files is discussed in issue #1982 and there are some ideas for making this possible through a new feature. If you want to comment on the subject, please visit issue.
```
Comment (danimtb) [inline docs/faqs.md:95]: Issue #1982 here is formatted automatically by Github markdown? I am not sure...

Comment (jgsogo) [reply docs/faqs.md:95]: Neither do I, but my plan was to try it and submit a fix if needed.

**Pull request merged**